electron/main/Store/storeConfig.ts

```diff
@@ -16,7 +16,7 @@ export interface LocalLLMConfig extends BaseLLMConfig {
   localPath: string;
 }
 
-export type LLMConfig = OpenAILLMConfig | LocalLLMConfig;
```

Comment:
This is probably causing all the build errors, do you want to clean up the rest of the code as well?